src/colorizer/Dataset.ts

```diff
@@ -182,6 +182,10 @@ export default class Dataset {
     return Object.keys(this.featureFiles);
   }
 
+  public get numObjects(): number {
+    return this.times?.length || this.trackIds?.length || this.features[this.featureNames[0]]?.data.length || 0;
```

Comment:
Is there one of these that you MUST have, to have a data set with items in it?  Just wondering why go through that chain.

Reply:
Hmm, features is probably the safest, since having a dataset without a feature doesn't make sense?